### ISSUE TITLE
Compatibility with tintin 2.02.02

### ISSUE THREAD

**Issue by KevinVanWorkum** [theixle/aardwolf-tintin#13]
I'm getting some errors with tintin 2.02.02.

```
#ERROR: PROMPT ROW IS INSIDE THE SCROLLING REGION: {Tick:29 QUEST: $quest_timer Exp: 850 Align:0} {3}.
debug: please use *GMCP[ROOM][INFO][exits][] instead of $GMCP[ROOM][INFO][exits][].

```

**Comment by theixle** [comment]
Hmm. That error message sounds familiar. Is it the resolution for the terminal emulator? If you have more lines/columns does it complain?

**Comment by Profpatsch** [comment]
I just bisected it:

Up to 20.01.8, everything is mostly fine (well, there is some overlap of `-` in the prompt which might lead to the error later?):

![screenshot](https://user-images.githubusercontent.com/3153638/91493188-3a1f2400-e8b7-11ea-95d8-ab0f42d8449d.png)

Then from 20.01.90:

![screenshot](https://user-images.githubusercontent.com/3153638/91493865-38099500-e8b8-11ea-9d9d-c862ba6f77fd.png)

**Comment by sbromberger** [comment]
Just upgraded and got the same thing. Has anyone found a workaround?

**Comment by sbromberger** [comment]
Just FYI, moving back to tintin++ v2.01.5 resolves the problem.

**Comment by jecxjo** [comment]
The issue is that `#showme` changed its format for handling splits. I'm working on Pull Request to fix this, will be up tonight.

**Comment by jecxjo** [comment]
Fix is in pull request #17

**Comment by jecxjo** [comment]
Fix was merged. Can someone else pull the latest and verify this is fixed?
